SYCL/BFloat16/bf16_storage_builtins.cpp

```diff
@@ -0,0 +1,444 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out -Xsycl-target-backend --cuda-gpu-arch=sm_80
+// RUN: %HOST_RUN_PLACEHOLDER %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+
+// Only cuda backend implements bf16
+// REQUIRES: cuda
+
+#include <CL/sycl.hpp>
+
+#include <cmath>
+#include <unordered_set>

```

Comment:
```suggestion
#include <vector>

```
I don't see uses of `std::unordered_set` here, but I see uses of `std::vector`.